### DISCUSSION TITLE
Will Sonoff RFBridge with Tasmota pick up more signals than without Tasmota

### DISCUSSION THREAD

**Discussion by SentiHassio1979** [arendst/Tasmota#10594] [Q&A] [answered]
After messing up one Sonoff RF Bridge with flashing to Tasmota, I have bought a new one. When I try to learn a remote with 433.92mhz before flashing with Tasmota nothing is learned. With knowing this how likely is it that the signal will be picked up after I have flashed this RF Bridge?

**Comment by barbudor** [comment, marked as answer]
After flashing tasmota, you have the possibility to even improve by changing the firmware on the secondary MCU.
By uploading "Portish" firmware, it will capture much more formats
However there is no guarantee. You may look at the doc of [Portish](https://github.com/Portisch/RF-Bridge-EFM8BB1) for more details.

What do you mean by "messing up" your Sonoff bridge ?

**Reply by SentiHassio1979** [reply]
When flashing tasmota.bin it does not connect with my wifi when pre entering the credentials and did not create a wifi network to connect for first time when not pre entering credentials. Blue light keeps on flashing.

**Reply by barbudor** [reply]
Do you have any special characters or space in your Wifi ?
Is it hidden ? 
You can come to the Tasmota discord server, in channel #flashing-issues to get more interactive help
I flashed my RFbridge without any issues

**Reply by SentiHassio1979** [reply]
Ok, sounds like I have quited to early trying with the first one :) will try more. Thank you very much!

**Comment by meingraham** [comment]
The Sonoff RF Bridge has two separate chips to handle the Wi-Fi (ESP8285) and RF (EFM8BB1) communications respectively. You flash Tasmota onto the ESP chip. To increase the types of RF devices detected by the Bridge, you also have to flash the RF ship with the Portisch firmware.

**Comment by SentiHassio1979** [comment]
Ok thank you both, will read more about it.

**Comment by PepeTheFroggie** [comment]
If you want to decode neighbors signals you better get a decent antenna. Replacing the 2 coil antennas with two 16.5cm long wires will do this. Yes, there are separate receive and transmit antennas.
